sfge/src/main/java/com/salesforce/graph/vertex/LiteralExpressionVertex.java

```diff
@@ -99,6 +117,10 @@ public void afterVisit(PathVertexVisitor visitor, SymbolProvider symbols) {
         public void afterVisit(SymbolProviderVertexVisitor visitor) {
             visitor.afterVisit(this);
         }
+
+        public static boolean isLiterallyFalse(BaseSFVertex vertex) {
+            return isExpectedLiteral(vertex, False.class, True.class);
+        }
     }
 
     public static final class True extends BooleanLiteralExpressionVertex {
```

Comment:
I suppose this question is out of the scope of this PR, but I wonder why the constructor sends the value of True as `Boolean.FALSE`.